dremio_toolkit/env_definition.py

```diff
@@ -36,23 +36,3 @@ class EnvDefinition:
     referenced_roles: list
     endpoint: str
 
-    def __init__(self, containers,
```

Comment:
You've removed this constructor and therefore python is complaining [here](https://github.com/ucesys/dremio-toolkit/actions/runs/4393649831/jobs/7694185802) that he cannot create EnvDefinition object with parameters.
```
E       TypeError: EnvDefinition() takes no arguments

dremio_toolkit/testing/mock_env_definition.py:25: TypeError
=========================== short test summary info ============================
FAILED dremio_toolkit/tests/test_create_snapshot.py::test_create_snapshot - AttributeError: 'EnvDefinition' object has no attribute 'containers'
FAILED dremio_toolkit/tests/test_env_file_writer.py::test_env_file_writer - TypeError: EnvDefinition() takes no arguments
FAILED dremio_toolkit/tests/test_env_reader.py::test_read_dremio_environment - TypeError: EnvDefinition() takes no arguments
============================== 3 failed in 0.18s ===============================
```